lib/javascript/index.js

```diff
@@ -51,16 +51,29 @@ module.exports = function(root, filePath, callback){
       })
       callback(error)
     }
+
     process.once('uncaughtException', exceptionHandler)
-    browserify(srcPath, {extensions: extensions}).transform('coffeeify').transform('babelify')
-    .on('error', exceptionHandler).bundle()
+
+    /**
+     * Pass all JS through browserify -> coffeeify -> babelify
+     */
+
+    browserify(srcPath, {extensions: extensions})
+    .transform('coffeeify', {bare: false, header: true})
+    .on('error', exceptionHandler)
```

Comment:
this line for the error handler causes different tests to fail, depending on where it's placed 🙁